### PR TITLE
[ADD] pos_disable_customer: new module to block customer creation in POS

### PR DESCRIPTION
**Pull request by suniagajose**
### [HU#1282](https://www.vauxoo.com/web#id=1282&view_type=form&model=user.story&menu_id=557&action=726)
- **CA#21** - Datos obligatorios para la facturación electrónica deben poder ser cargados desde el POS

Timesheet in [Vx#6109](https://www.vauxoo.com/web#id=6109&view_type=form&model=project.task&menu_id=136&action=138)

**Disable button to create customer**
odoo pos

### CODE REVIEW

**Comment by sabrinagnu** [comment]
Functional tests 👍

File: pos_disable_customer/static/src/js/pos_disable_customer.js [hunk 1]
```diff
@@ -0,0 +1,14 @@
+odoo.define('pos_disable_customer.pos_disable_customer', function (require){
+"use strict";
+
+var screens = require('point_of_sale.screens');
+
+    screens.ClientListScreenWidget.include({
+        renderElement: function(){
+            this._super();
+            this.$('.new-customer').hide();
+            console.log(this.$('.new-customer'));
```
Comment (hugho-ad) [inline pos_disable_customer/static/src/js/pos_disable_customer.js]: Small Coment
- Remove console.log

Comment (suniagajose) [reply pos_disable_customer/static/src/js/pos_disable_customer.js]: you right!

thanks! @hugho-ad

**Comment by hugho-ad** [comment]
@suniagajose 
This feature coul be in the client app, what do you think ?

For me +1

**Comment by suniagajose** [comment]
hi @hugho-ad 

i think that this feature can be use fot othe,rs client...

but i am open to opinions

regards

**Comment by hugho-ad** [comment]
@suniagajose 
It's a very small feature, this will be an extra dependency in the client app, for this i think that may be this must be in the client app

This could be migrated  in other moment,

:+1:

**Comment by suniagajose** [comment]
Following your advice @hugho-ad was moved this feature to client repo

This PR will be closed.

Regards,

**Pull request closed without merging**